agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,25 +15,17 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private final ArrayList<String> mssgs = {"Hello visitor!", "I am here!", "Go away."};
-
   @Override
-  public String doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    /** 
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello AnneMarie!</h1>");
-    */
-    Gson gson = new Gson();
-    String json = gson.toJson(mssgs);
-    return json;
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // other branches have code here; however, this one does not.
```

Comment:
Is the end goal to remove this servlet altogether?

Reply:
This just got branched before this class worked properly. I can just remove it from the commit entirely, which I think would be best, because the servlet is fully-functioning and necessary to the comment function just on a different branch.

Reply:
Done